vendor/assets/stylesheets/the-event/style.css

```diff
@@ -11,19 +11,19 @@
 
 body {
   background: #fff;
-  color: #2f3138;
+  color: #fff;
```

Comment:
Please move the relevant ones to app/assets/ (and separates different components into different files in the home-components/ folder) instead of editing vendor/assets/, same for the other changes in this file, because we want to filter out the unused styling